src/WebJobs.Script/Rpc/ILanguageWorkerChannelManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Diagnostics;
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc
+{
+    public interface ILanguageWorkerChannelManager
+    {
+        Task InitializeWebhostChannelsAsync(IEnumerable<string> languages);
```

Comment:
Can this just be called InitializeChannelsAsync? Removing the WebHost naming. Also, can we have just one initialize method taking a single language - caller can call multiple in parallel if needed.

Reply:
Makes send to just keep one initialize method.  This method is specifically for initializing language worker channels at webhost level